src/redux/books/books.js

```diff
@@ -1,25 +1,34 @@
 const ADD_BOOK = 'bookStore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+const LOAD_BOOKS = 'bookStore/books/LOAD_BOOKS';
 
 const initialState = [];
 
 export const addBook = (payload) => ({
   type: ADD_BOOK,
   payload,
 });
+
+export const loadBooks = (payload) => ({
+  type: LOAD_BOOKS,
+  payload,
+});
+
 export const removeBook = (id) => ({
   type: REMOVE_BOOK,
   id,
 });
-const reducer = (state = initialState, action) => {
+
+const booksReducer = (state = initialState, action) => {
   switch (action.type) {
     case ADD_BOOK:
       return ([...state, action.payload]);
+    case LOAD_BOOKS:
+      return (action.payload);
     case REMOVE_BOOK:
-      return (state.filter((book) => book.id !== action.id));
-    default:
-      return state;
+      return state.filter((book) => book.id !== action.id);
+    default: return state;
   }
 };
 
-export default reducer;
+export default booksReducer;
```

Comment:
[OPTIONAL] The books reducer is not following the ducks pattern, this pattern is useful to reuse code in later stages, facilitate imports and keep the reducer logic on a single file. [Check the template below and you can read more about it here.](https://github.com/erikras/ducks-modular-redux)

![image](https://user-images.githubusercontent.com/79114233/145428191-edec102b-a371-4873-9361-873732525bae.png)